lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1513,13 +1513,14 @@ apos.define('apostrophe-schemas', {
     self.addFieldType({
       name: 'color',
       populate: function (data, name, $field, $el, field, callback) {
-        $field.val(data[name]);
-        apos.ui.enhanceColorpicker($field);
+        var options = field.spectrumOptions || {};
+        apos.ui.enhanceColorpicker($field, data[name], options);
         return setImmediate(callback);
       },
       convert: function (data, name, $field, $el, field, callback) {
         if ($field.spectrum('get')) {
-          data[name] = $field.spectrum('get').toHexString();
+          data[name] = $field.spectrum('get');
+          data[name].stringFormat = $field.val();
```

Comment:
This is a bc break. Consumers are not expecting to have to treat this data differently all of a sudden in their templates. I would be comfortable with any string that is valid in the same places the old string was valid as far as the browser is concerned. Is that not possible?

Reply:
I backed out this idea of of storing the full tinycolor obj instead of the string version, ultimately didn't feel necessary. testing both a sandbox and an advanced implementation upgrades seamlessly (i do need twin apostrophe-palette update thought)